public/index.css

```diff
@@ -368,4 +370,92 @@ img {
 }
 .btn--icon:active img {
   transform: scale(0.9);
-}
\ No newline at end of file
+}
+
+.autumn,
+.summer,
+.winter {
+  display: none;
+}
+
+.autumn-bg .autumn,
+.summer-bg .summer,
+.winter-bg .winter {
+  display: block;
+}
+
+#background {
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  margin: auto;
+  left: 0;
+  top: 0;
+  top: 0;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.ftm-mountain {
+  bottom: -5px;
+  position: absolute;
+  left: -75px;
+  right: 0;
+  margin: auto;
+  width: 150%;
+  z-index: 2;
+}
+
+.ftm-totem {
```

Comment:
@ashwinnair-chimple instead of making media query I use values for bottom and left position as `vw` this way we can eliminate or lessen extra code in css. but pls tell me know if you're okay with this. I checked the tablet and mobile the UI are consistent.

Reply:
thats okay, all we need is the app should be responsive.